config/defaults.hjson

```diff
@@ -73,7 +73,7 @@
     debug: false
   }
   # Pictrs image server configuration.
-  pictrs_config: {
+  pictrs: {
     # Address where pictrs is available (for image hosting)
     url: "http://pictrs:8080/"
```

Comment:
According to #2359 this one shouldnt have the trailing slash, but its added automatically by Url. So only way to avoid this would be to change the type back to String.